zrml/prediction-markets/src/lib.rs

```diff
@@ -1281,9 +1315,16 @@ mod pallet {
                     )?;
                 }
 
+                // TODO(#372): Allow court with global disputes.
                 // ensure, that global disputes controls the resolution now
                 // it does not end after the dispute period now, but after the global dispute end
-                Self::remove_last_dispute_from_market_ids_per_dispute_block(&disputes, &market_id)?;
+                if let Some(auto_resolve_block) =
+                    T::SimpleDisputes::get_auto_resolve(&disputes, &market_id, &market)?
```

Comment:
https://github.com/zeitgeistpm/zeitgeist/blob/8a8c4b954b3f7ca1f215edc4f99ef5cf44d559df/zrml/prediction-markets/src/lib.rs#L1286 

Yeah not clearly visible, but it is ensured above.

> The check for an auto-resolve is already in clear_auto_resolve.

I don't get this one. I need to clear the auto resolution, because global dispute controls the resolution then.

Reply:
Yeah, that is what we can do here 💯 noice. You just also reminded me to add benchmarks components here.

Reply:
cb038d8a97aaa2e9cc58e6747cfc16a3e2590b75